src/ssr-template-helper.js

```diff
@@ -80,23 +80,34 @@ export class SsrTemplateHelper {
     if (!opt_templates) {
       mustacheTemplate = this.templates_.maybeFindTemplate(element);
     }
-    return this.viewer_.sendMessageAwaitResponse(
-      'viewerRenderTemplate',
-      this.buildPayload_(
-        request,
-        mustacheTemplate,
-        opt_templates,
-        opt_attributes
-      )
-    );
+    return this.viewer_
+      .isTrustedViewer()
+      .then(trusted => {
+        userAssert(
+          trusted,
+          'Refused to apply SSR in untrusted viewer: ',
```

Comment:
Pedantic: Remove "apply" which applies to the other method.

Reply:
I already had that fix in a followup commit! you are quick (and I should slow down with my pushes)